spec/checkPullRequestReviewSpec.js

```diff
@@ -1,11 +1,8 @@
 // Copyright 2020 The Oppia Authors. All Rights Reserved.
-//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-//
 //      http://www.apache.org/licenses/LICENSE-2.0
-//
```

Comment:
Unneeded change.

Reply:
Correcting it in the latest commit Sir.Thanks.